src/timelines/preamble.js

```diff
@@ -1,28 +1,29 @@
-import experimentStart from '../trials/experimentStart'
-import startCode from '../trials/startCode'
-import instructions from '../trials/instructions'
-import holdUpMarker from '../trials/holdUpMarker'
-import { AT_HOME, VIDEO } from '../config/main'
-import adjustVolume from '../trials/adjustVolume'
-import camera from '../trials/camera'
+import experimentStart from "../trials/experimentStart";
+import startCode from "../trials/startCode";
+import instructions from "../trials/instructions";
+import holdUpMarker from "../trials/holdUpMarker";
+import { USE_EVENT_MARKER, VIDEO, VOLUME } from "../config/main";
+import adjustVolume from "../trials/adjustVolume";
+import camera from "../trials/camera";
 
-console.log('at_home', AT_HOME)
-console.log('env at home', process.env.REACT_APP_AT_HOME)
 
-let tl
-if (AT_HOME && !VIDEO) {
-  tl = [experimentStart(), instructions()]
-}
-else if (AT_HOME && VIDEO) {
-  tl = [experimentStart(), instructions(), camera()]
-}
-else {
-  tl = [experimentStart(), adjustVolume(), holdUpMarker(), startCode(), instructions()]
+let tl = [experimentStart()];
+if (VIDEO) {
+  tl.push(instructions());
```

Comment:
I'd make this longer and more explicit to make it easier to follow the logic.  So an `if` for each thing that is dependent on config.  Looking at the old code.  We should check volume first, then event marker for `holdUpMarker` and `startCode` then just push instructions, then check for `VIDEO` for `camera`